docs/src/userguide/external_C_code.rst

```diff
@@ -97,8 +97,8 @@ match the C ones, and in some cases they shouldn't or can't. In particular:
 
 #. If the header file uses macros to define constants, translate them into a
    normal external variable declaration.  You can also declare them as an
-   :keyword:`enum` if they contain normal :c:type:`int` values.  Note that
-   Cython considers :keyword:`enum` to be equivalent to :c:type:`int`, so do
+   :keyword:`enum` if they contain normal :c:expr:`int` values.  Note that
+   Cython considers :keyword:`enum` to be equivalent to :c:expr:`int`, so do
```

Comment:
Shouldn't this use `:c:type:`?

Reply:
When we use `:c:type:` sphinx is trying to find definition of `int` type which does not exist. This yields warnings during generating of documentation:
```
/Users/matus/dev/cython/docs/src/userguide/external_C_code.rst:98: WARNING: Unparseable C cross-reference: 'int'
Invalid C declaration: Expected identifier in nested name, got keyword: int [error at 3]
  int
  ---^
``` 
The fix is to use `c:expr:` instead - see https://stackoverflow.com/questions/63080103/how-to-reference-built-in-c-types-with-rst for details.